<launch>
  <include file="$(find robonomics_control)/launch/control.launch" />
  <include file="$(find robonomics_liability)/launch/liability.launch" />

  <node ns="opcua" name="opcua_client" pkg="ros_opcua_impl_freeopcua" type="client_node" output="screen" />

  <group ns="welder">
    <param name="actionlib_server_sub_queue_size" value="50" />

    <node pkg="robonomics_game_plant" type="plant_node.py" name="plant" output="screen">
      <param name="opcua_endpoint" value="opc.tcp://192.168.10.3:53880/" />
      <param name="opcua_server_namespace" value="ns=3;s=/Airalab/Plant1" />
    </node>

    <node pkg="robonomics_game_supply" type="supply_node" name="supply_chain" output="screen">
      <param name="liability_node" value="/liability" />
      <param name="plant_node" value="plant" />
      <param name="plant_addr" value="1" />
    </node>

    <include file="$(find robonomics_market)/launch/market.launch" />
  </group>
</launch>
